isobus/src/can_network_manager.cpp

```diff
@@ -281,6 +285,39 @@ namespace isobus
 		CANNetworkManager::CANNetwork.receive_can_message(tempCANMessage);
 	}
 
+	void CANNetworkManager::on_partner_deleted(PartneredControlFunction *partner, CANLibBadge<PartneredControlFunction>)
+	{
+		CANStackLogger::CAN_stack_log("[NM]: A partner was deleted.");
```

Comment:
Maybe add information which partner was deleted, e.g:
```suggestion
		CANStackLogger::CAN_stack_log("[NM]: Partner " + isobus::to_string(static_cast<int>(partner->get_address()))  + " was deleted.");
```